token_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("token", func() {
 	Context("tokenMan", func() {
 
 		It("should get a valid token", func() {
-
+log.Info("\n\n\n\nHERE\n\n")
```

Comment:
Mitch, This log was perhaps used for debugging. Is this MR fixing the race issue?